src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/GetUnallocatedCaseService.kt

```diff
@@ -46,15 +44,12 @@ class GetUnallocatedCaseService(
       val requirements = communityApiClient.getActiveRequirements(crn, convictionId)
       val courtReportDocuments = getDocuments(crn, convictionId)
 
-      val assessment = assessmentApiClient.getAssessment(crn)
-        .map { assessments -> Optional.ofNullable(assessments.maxByOrNull { a -> a.completed }) }
-      val results = Mono.zip(offenderSummary, requirements, courtReportDocuments, assessment, conviction).block()!!
+      val results = Mono.zip(offenderSummary, requirements, courtReportDocuments, conviction).block()!!
       val age = Period.between(results.t1.dateOfBirth, LocalDate.now()).years
       return UnallocatedCaseDetails.from(
-        it, results.t1, age, results.t5?.offences,
-        results.t5?.sentence?.expectedSentenceEndDate, results.t5?.sentence?.description, results.t2.requirements,
-        results.t3,
-        results.t4.orElse(null)
+        it, results.t1, age, results.t4?.offences,
+        results.t4?.sentence?.expectedSentenceEndDate, results.t4?.sentence?.description, results.t2.requirements,
+        results.t3
```

Comment:
Is there no need for the orElse(null) here any more?

Reply:
its the silly naming of fields again, t5 became t4 after removing getting the assessment so no need for the orElse(null)